src/stream/src/executor/actor.rs

```diff
@@ -101,6 +108,32 @@ impl ActorContext {
     }
 }
 
+tokio::task_local! {
+    /// TODO: report errors and prompt the user.
+    static ACTOR_ERRORS: RefCell<HashMap<String,Vec<ExprError>>>;
```

Comment:
I think `task_local` or (`thread_local` in non-async env) is always not a best practice and would like to avoid it as far as possible. Imaginate others also use task-local variables everywhere to pass in/out something, the code will become hard to maintain and you never know where a variable is set, get or cleaned.

Please consider simply passing a context struct to each executor from actor. It could also be used to pass configurations in the future.

Reply:
It seems like we can also put `actor_id` in the context. We may have to use `Mutex` to pass the compilation, but it'd be better to wrap it on inner fields instead of the whole context.